packages/react/src/button-group/ButtonGroup.css.ts

```diff
@@ -0,0 +1,85 @@
+import { type RecipeVariants, createVar, recipe } from "../vanilla-extract";
+const roundedBorderRadius = createVar();
+export const buttonGroup = recipe({
+  base: {
+    borderRadius: "0",
+    vars: {
+      [roundedBorderRadius]: "5px",
```

Comment:
let's do the opposite - so instead of setting the border-radius of the outer corners let's set the border-radius of the inner elements to `0`

Reply:
It is actually doing that. Let me rename the 'buttonGroup' to 'button' and 'parentButtonGroup' to 'buttonGroup' for better understandability.

Reply:
@JafarSaifullah how do I render a button group without any border radius?

Reply:
```diff
diff --git a/packages/react/src/button-group/ButtonGroup.css.ts b/packages/react/src/button-group/ButtonGroup.css.ts
index 6f63c47..5525ed3 100644
--- a/packages/react/src/button-group/ButtonGroup.css.ts
+++ b/packages/react/src/button-group/ButtonGroup.css.ts
@@ -1,17 +1,18 @@
-import { type RecipeVariants, createVar, recipe } from "../vanilla-extract";
-const roundedBorderRadius = createVar();
+import { type RecipeVariants, recipe } from "../vanilla-extract";
+
 export const button = recipe({
   base: {
-    borderRadius: "0",
-    vars: {
-      [roundedBorderRadius]: "5px",
+    selectors: {
+      "&:not(:first-child):not(:last-child)": {
+        borderRadius: 0,
+      },
     },
   },
   compoundVariants: [
     {
       style: {
         selectors: {
-          ["&:not(:first-child)"]: {
+          "&:not(:first-child)": {
             borderLeft: "none",
           },
         },
@@ -24,7 +25,7 @@ export const button = recipe({
     {
       style: {
         selectors: {
-          ["&:not(:first-child)"]: {
+          "&:not(:first-child)": {
             borderTop: "none",
           },
         },
@@ -39,25 +40,25 @@ export const button = recipe({
     orientation: {
       horizontal: {
         selectors: {
-          [`&:first-child`]: {
-            borderBottomLeftRadius: roundedBorderRadius,
-            borderTopLeftRadius: roundedBorderRadius,
+          "&:first-child": {
+            borderBottomRightRadius: 0,
+            borderTopRightRadius: 0,
           },
-          [`&:last-child`]: {
-            borderBottomRightRadius: roundedBorderRadius,
-            borderTopRightRadius: roundedBorderRadius,
+          "&:last-child": {
+            borderBottomLeftRadius: 0,
+            borderTopLeftRadius: 0,
           },
         },
       },
       vertical: {
         selectors: {
-          [`&:first-child`]: {
-            borderTopLeftRadius: roundedBorderRadius,
-            borderTopRightRadius: roundedBorderRadius,
+          "&:first-child": {
+            borderBottomLeftRadius: 0,
+            borderBottomRightRadius: 0,
           },
-          [`&:last-child`]: {
-            borderBottomLeftRadius: roundedBorderRadius,
-            borderBottomRightRadius: roundedBorderRadius,
+          "&:last-child": {
+            borderTopLeftRadius: 0,
+            borderTopRightRadius: 0,
           },
         },
       },
```

Reply:
Understood and addressed bhai